Automattic/jetpack
# Instant Search: Add product ratings to search results

#### Changes proposed in this Pull Request:
* Fixes #18403.
* Render product ratings for the currently unreleased Product search layout.
* Rearrange the Product search result component.
* Tweaks ESLint configuration for the Jetpack Search module to accommodate the `jsx` pragma for Preact.

#### Jetpack product discussion
None.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:

* Apply this change to your Jetpack site with WooCommerce products. You may need to rate some of your products to see the result of this PR.
* Perform a search in the product search layout by appending `result_format=product` to the query string.

#### Proposed changelog entry for your changes:
* None. (Product search layout has not yet been released to the public.)

#### Visual changes:
After: 
<img width="707" alt="Screen Shot 2021-01-12 at 2 25 34 PM" src="https://user-images.githubusercontent.com/4044428/104376293-0baa8a00-54e2-11eb-8b9e-7162db7dfbaf.png">

Before:
<img width="707" alt="Screen Shot 2021-01-12 at 2 29 51 PM" src="https://user-images.githubusercontent.com/4044428/104376751-b4f18000-54e2-11eb-823d-9e8911166ae9.png">
